codegen/templates/code/controller/controller.gotmpl

```diff
@@ -1,3 +1,5 @@
+// Code generated by autopilot. DO NOT EDIT.
+
```

Comment:
can we append this header here instead: https://github.com/solo-io/autopilot/blob/dev/codegen/writer/writer.go#L47 (after the writer checks that the file is .go)